crates/pubnub/src/pubnub_client.rs

```diff
@@ -0,0 +1,4 @@
+//! TODO: Add documentation
+
+/// TODO: Add documentation
```

Comment:
We have to remember to provide the description for the modules in the meanwhile. ;d